hazelcast-code-generator/src/main/java/com/hazelcast/client/protocol/generator/CodeGenerationUtils.java

```diff
@@ -56,6 +56,30 @@
         put("com.hazelcast.map.impl.SimpleEntryView", "SimpleEntryView");
     }};
 
+    private static final Map<String, String> JAVA_TO_NODE_TYPES = new HashMap<String, String>() {{
+        put(DATA_FULL_NAME, "data");
+        put("java.lang.String", "string");
+        put("java.lang.Integer", "int32");
+        put("boolean", "boolean");
+        put("int", "int32");
+        put("com.hazelcast.nio.Address", "Address");
+        put("java.util.List", "list");
+        put("java.util.Set", "set");
+    }};
+
+    private static final Map<String, String> JAVA_TO_TS_TYPES = new HashMap<String, String>() {{
```

Comment:
@bilalyasar inconsistency, `JAVA_TO_TS_TYPES`. We need to stick either `TS` or `JS`. `TS` makes more sense because we're producing typescript output